### PR TITLE
doc/rados: m-config-ref: edit "background"

### PR DESCRIPTION
**Pull request by zdover23**
Edit the "Background" section of doc/rados/monitor/config-ref.rst

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/configuration/mon-config-ref.rst [hunk 1]
```diff
@@ -16,24 +16,30 @@ consistent, but you can add, remove or replace a monitor in a cluster. See
...
-The primary role of the Ceph Monitor is to maintain a master copy of the cluster
-map. Ceph Monitors also provide authentication and logging services. Ceph
-Monitors write all changes in the monitor services to a single Paxos instance,
-and Paxos writes the changes to a key/value store for strong consistency. Ceph
-Monitors can query the most recent version of the cluster map during sync
-operations. Ceph Monitors leverage the key/value store's snapshots and iterators
-(using leveldb) to perform store-wide synchronization.
+Ceph OSD Daemons or Ceph Metadata Servers, they must connect to a Ceph Monitor.
+When a Ceph client has a current copy of the cluster map and the CRUSH
+algorithm, it can compute the location for any object. This ability to compute
```
Comment (anthonyeleven) [inline doc/rados/configuration/mon-config-ref.rst:27]: for any RADOS object within the cluster.

File: doc/rados/configuration/mon-config-ref.rst [hunk 2]
```diff
@@ -16,24 +16,30 @@ consistent, but you can add, remove or replace a monitor in a cluster. See
...
+the locations of objects makes it possible for Ceph Clients to talk directly to
+Ceph OSD Daemons. This direct communication with Ceph OSD Daemons represents an
+improvment upon traditional storage architectures in which clients were
+required to communicate with a central component, and that improvment
+contributes to Ceph's high scalability and performance. See `Scalability and
+High Availability`_ for additional details.
+
+The Ceph Monitor's primary function is to maintain a master copy of the cluster
+map, but Ceph Monitors have ancillary functions of providing authentication and
```
Comment (anthonyeleven) [inline doc/rados/configuration/mon-config-ref.rst:36]: suggest Monitors also provide auth...

**Review by anthonyeleven — APPROVED** [review]
Several wording suggestions

File: doc/rados/configuration/mon-config-ref.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/configuration/mon-config-ref.rst:27]: That's good.

File: doc/rados/configuration/mon-config-ref.rst (hunk 2 above)
Comment (zdover23) [reply doc/rados/configuration/mon-config-ref.rst:36]: I was wondering about that, and I will defer to your fresh eyes.

Comment (anthonyeleven) [reply doc/rados/configuration/mon-config-ref.rst:36]: Just seems a bit less stuffy

Comment (zdover23) [reply doc/rados/configuration/mon-config-ref.rst:36]: I agree.

File: doc/rados/configuration/mon-config-ref.rst (hunk 1 above)
Comment (anthonyeleven) [reply doc/rados/configuration/mon-config-ref.rst:27]: "object" means so many things in Ceph that I try to be clear about RADOS objects vs S3 objects vs omap.

Comment (zdover23) [reply doc/rados/configuration/mon-config-ref.rst]: This is a disambiguating directive that we will all hereafter follow.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/51272 - Reef backport
https://github.com/ceph/ceph/pull/51273 - Quincy backport
https://github.com/ceph/ceph/pull/51274 - Pacific backport

**Pull request merged**